docker-compose.yml

```diff
@@ -0,0 +1,3 @@
+app:
+  build: .
+  
```

Comment:
Throws an error

ERROR: Cannot locate specified Dockerfile: Dockerfile

Reply:
we already have the Dockerfile in root path